scripts/build/Dockerfile.tmpl

```diff
@@ -21,17 +21,24 @@ RUN apt-install \
 	libprotobuf-c-dev \
 	libprotobuf-dev \
 	libselinux-dev \
+	iproute2 \
 	pkg-config \
 	protobuf-c-compiler \
 	protobuf-compiler \
 	python3-minimal \
+	python3-protobuf \
+	python3-yaml \
 	python3-future
 
 COPY . /criu
 WORKDIR /criu
 ENV $ENV1=yes
 
-RUN make mrproper && date && \
+# zdtm uses an unversioned python binary to run the tests.
+# let's point python to python3
+RUN  ln -sf /usr/bin/python3 /usr/bin/python
```

Comment:
On my Ubuntu VM 18.04 a link from python to python3 is provided by the package python-minimal. Would that also work here?

Reply:
You are right, the `python-minimal` package includes [`/usr/bin/python`](https://packages.ubuntu.com/bionic/amd64/python-minimal/filelist), but this package has been dropped in Ubuntu (Focal). The replacement package `python3-minimal` provides only [`/usr/bin/python3`](https://packages.ubuntu.com/focal/amd64/python3-minimal/filelist)

Reply:
I see there exists a package called python-is-python3. Does that help?

Reply:
That's perfect, thank you.